neofetch

```diff
@@ -2178,6 +2178,32 @@ END
                          "${XDG_CONFIG_HOME}/sakura/sakura.conf")"
         ;;
 
+        "st")
+            if term_font="$(ps -o command= -p "$parent" | grep -E '\-f\s?.*?:?')"; then
+                term_font="${term_font/*-f/}"
+                term_font="${term_font/ -*/}"
+            else
+                binary=""
+
+                # On Linux we can get the exact path to the running binary through the procfs
+                # on other systems we just have to guess and assume `st` is invoked from somewhere
+                # in the users $PATH
+                [[ -L /proc/$parent/exe ]] && binary="/proc/$parent/exe"
+                [[ -z "$binary" ]] && binary="$(command -v st)"
```

Comment:
We know `$binary` will be unset/empty so we can remove the test.

Also, change `command -v` to `type -p` for consistency with the rest of the script.

Reply:
I've changed this as follows:

```bash
[[ -L /proc/$parent/exe ]] && binary="/proc/$parent/exe" || binary="$(type -p st)"
```